Library/Homebrew/utils/pypi.rb

```diff
@@ -332,6 +333,8 @@ def self.update_python_resources!(formula, version: nil, package_name: nil, extr
       EOS
     end
 
+    odie "Excluded superfluous packages: #{", ".join(exclude_packages)}" if exclude_packages.any?
```

Comment:
Damned, I've done too much python recently
```suggestion
    odie "Excluded superfluous packages: #{exclude_packages.join(", ")}" if exclude_packages.any?
```